lib/cartopy/_crs.pyx

```diff
@@ -133,16 +126,21 @@ cdef class CRS:
     """
     def __init__(self, proj4_params, globe=None):
         """
-        Parameters
-        ----------
-        proj4_params : iterable of key-value pairs
-            The proj4 parameters required to define the desired CRS.
-            The parameters should not describe the desired elliptic model,
-            instead create an appropriate Globe instance. The ``proj4_params``
-            parameters will override any parameters that the Globe defines.
-        globe : :class:`~cartopy.crs.Globe` instance, optional
-            If omitted, the default Globe instance will be created.
-            See :class:`~cartopy.crs.Globe` for details.
+        Args:
```

Comment:
Please target numpydoc format (https://numpydoc.readthedocs.io/en/latest/) wherever possible.

Reply:
That was the form of the cartopy docs previously, but they weren't rendering properly with the use of sphinx-gallery.  I've spent a lot of hours changing them all to be not in this format, but more like the Iris docs.